WordPress/twentytwentyfive
# Personal Blog Templates: Text-only alternative

**Description**

Adds Personal Blog templates (Text-only alternative) listed in https://github.com/WordPress/twentytwentyfive/issues/132
This PR includes the file changes from https://github.com/WordPress/twentytwentyfive/pull/138, as patterns.

**Screenshots**
Home:
![text-only-blog-home](https://github.com/user-attachments/assets/6c7a9be7-2a23-48d9-babd-d4a44e96d3ce)

![text-only-blog-home-small](https://github.com/user-attachments/assets/cec30da2-38ab-408f-a356-aa1e214c793a)

Archive:

![text-only-blog-archive](https://github.com/user-attachments/assets/4bb5fdff-93d3-4589-96c3-fb7f94d3b9c8)

Single:

![text-only-blog-single](https://github.com/user-attachments/assets/da2dde47-2a2b-4f94-be58-0c776055efcb)

Search:
![text-only-blog-search](https://github.com/user-attachments/assets/fbd1a3d9-8c77-4d9c-919d-73f02a058cf5)


**Testing Instructions**
Activate Gutenberg.
Apply the PR.
On your WordPress test installation, make sure that you have enough content to trigger the query pagination and the next and previous post navigation.

Go to Appearance > Editor and open the Home template to edit it.
In the "Template" panel in the sidebar, open the "Design" panel and select the "text-only blog, home" template.
Save.
Confirm that the design matches the design on Figma, both in the editor and front.

Now repeat the same test for the archive, search, and single post templates.
